### PR TITLE
fixed README and switched requrements.txt's place to the main folder

### PR DESCRIPTION
**Pull request by devshaki**
# Pull Request Template

## Description

Please briefly describe the purpose of this pull request.

## Changes Made

List the main changes and improvements introduced by this PR.

- Change 1
- Change 2
- ...

## Screenshots (if applicable)

If your changes include visual modifications, please attach relevant screenshots here.

## Checklist

Please make sure that your pull request meets these requirements before submission:

- [ ] The code follows the project's coding standards and style guidelines.
- [ ] Tests have been added or updated to cover the changes.
- [ ] Documentation (if applicable) has been updated to reflect the changes.
- [ ] The code has been reviewed and any issues have been addressed.
- [ ] The PR title is descriptive and follows the project's naming conventions.

## Related Issues

Link any related issues here using the format `Fixes #123` or `Closes #456`.

## Checks

Before merging, the following checks should pass:

- [ ] Code review has been completed.
- [ ] All tests pass successfully.
- [ ] Documentation (if applicable) has been reviewed and updated.
- [ ] The PR has at least one approval from a maintainer or reviewer.
- [ ] Any relevant issues have been linked and addressed.
- [ ] I checked for issues arising after run

### CODE REVIEW

File: requirements.txt
Comment (prasad-chavan1) [inline requirements.txt:1]: What changes you have done here ???

Comment (devshaki) [reply requirements.txt:1]: you didnt say how to install the libraries from the requrement.txt, plus i moved the requrement.txt to the main folder bacause its a bit nicer(rlly no need to do it i just think its nicer)

Comment (prasad-chavan1) [reply requirements.txt:1]: revert that file back and just give ss of updated readme ill merge

**Comment by devshaki** [comment]
image

File: requirements.txt
Comment (devshaki) [reply requirements.txt:1]: alr

**Comment by prasad-chavan1** [comment]
@devshaki  Great Job 🤟🏻

**Comment by prasad-chavan1** [comment]
Open More issues and Contribute

**Pull request merged**